docs/namespaces-scaling-mercury.md

```diff
@@ -0,0 +1,91 @@
+
+# Namespaces and Scaling in Mercury
+[TOC]
+
+## Structure
+
+Broadly speaking, the Mercury RChain network is a directed acyclic graph ("dag") labeled by complete lattices.  The **organizational dag** is a rooted dag formed by superimposing a collection of trees.  Each **organizational tree** in the collection has a collection of vertices, called **locales**; locales may be shared between different organizational trees.  Each locale has a collection of **regions** that form the basis of a complete lattice of **regional namespaces**.  Each regional namespace has a countably infinite collection of **names** formed by quoting Rholang processes.
+
+[[ graphics here ]]
+![](https://cdn.monetizejs.com/resources/button-32.png)
```

Comment:
monetize.js?

Reply:
Ah!  Thanks: I'd grabbed the example image form stackedit.io and didn't update.  Fix coming.